src/index.css

```
@@ -1,6 +1,8 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import "tailwindcss";
+@import "tw-animate-css";
+@config "../tailwind.config.ts";
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@config`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)